apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-tabbed-example.component.html

```diff
@@ -228,6 +228,13 @@
                     debitis. Similique vel ipsam debitis fuga sequi eveniet perspiciatis?
                     <br />
                     <br />
+                    <button
+                        fd-button
+                        (click)="switchTab('tab_1')"
+                        title="Switch  to tab 1"
```

Comment:
`Switch  to tab 1` -> `Switch to tab 1`

Reply:
did not notice the extra space 🤦 thanks 😄